static/src/js/components/KeywordRecommendationsKeyword/KeywordRecommendationsKeyword.jsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Do not use `eslint-disable`s for the entire file. If you absolutely need them, you should use `eslint-disable-line` for each individual case, leaving a comment why it was used if its not obvious.